src/db/db.ts

```diff
@@ -12,6 +12,58 @@ export async function addAlice() {
   await closeConnection();
 }
 
+export async function createUsers(user: users) {
+  await prisma.users.create({
+    data: {
+      usersname: user.usersname,
+      usersemail: user.usersemail,
+      userspassword: user.userspassword,
+      userstotalscore: null,
+      userslevel: null
+    }
+  });
+  await closeConnection();
+}
+
+export async function getAllUsers() {
+  const users = await prisma.users.findMany();
+  console.log('This is printing out all users', users);
+
+  await closeConnection();
+}
+
+export async function getUserByID(id: number) {
```

Comment:
This needs to return the user object

Reply:
I am gona fix this. Thank you for review.